tests/unit/inference/test_inference.py

```diff
@@ -341,6 +341,46 @@ def test(
         assert assert_fn(bs_output, ds_output)
 
 
+# Setup for these models is different from other pipelines, so we add a separate test
+@pytest.mark.inference
+class TestStableDiffusion(DistributedTest):
+    world_size = 1
+
+    def test(self):
+        from diffusers import DiffusionPipeline
+        from image_similarity_measures.evaluate import evaluation
```

Comment:
Make sure to add this to the requirements somewhere

Reply:
Could add this to requirements-sd?

Reply:
Added to requirements-sd, thanks.